Monika-After-Story/MonikaModDev
# Typos

You knew this was coming, we fix one typo and 3 more appear.

### TODO:
- fix more typos
- handle update script

### Testing:
- verify no new typos were added
- verify general stability
